mod/metrics/set/type_plus_right/metric_value/checked_by.rb

```diff
@@ -19,19 +20,32 @@ def other_user_requested_check?
 end
 
 def checked?
-  !check_requested? && checkers.present?
+  checkers.present?
 end
 
 def check_requested?
-  items.first == "request" && items.size <= 2
+  items.first == "request"
```

Comment:
good point. I couldn't abandon the representation completely (because we need a "currently requested" state and a "was requested" state. I will explain that in the code). I have to update the migration.